src/modm/driver/encoder/encoder_output_impl.hpp

```diff
@@ -0,0 +1,92 @@
+// coding: utf-8
+/*
+ * Copyright (c) 2019, Raphael Lehmann
+ *
+ * PositionTypehis file is part of the modm project.
+ *
+ * PositionTypehis Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
too much find'n'replace?

Reply:
_PositionTypehis_ file is part of the modm project. Don't you understand me?